lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/LibertyWorkspace.java

```diff
@@ -62,8 +82,54 @@ public List<Feature> getInstalledFeatureList() {
         return this.installedFeatureList;
     }
 
-    public void setInstalledFeatureList(List<Feature> installedFeatureList){
+    public void setInstalledFeatureList(List<Feature> installedFeatureList) {
         this.installedFeatureList = installedFeatureList;
     }
 
+    public void initConfigFileList() {
+        try {
+            List<Path> configPathsList = Files.find(Paths.get(getWorkspaceURI()), Integer.MAX_VALUE, (filePath, fileAttributes) -> 
+                    LibertyUtils.isServerXMLFile(filePath.toString()) || (fileAttributes.isDirectory() && filePath.toString().endsWith("configDropins")))
```

Comment:
As we discussed, should add `configDropins/overrides`, `configDropins.defaults` and `usr/shared/config` here.